gpu/kinfu/tools/kinfu_app_sim.cpp

```diff
@@ -1078,47 +1078,46 @@ struct KinFuApp
       PtrStepSz<const KinfuTracker::PixelRGB> rgb24_sim = PtrStepSz<const KinfuTracker::PixelRGB>(height, width, color_buf_, width);
       tic_toc.push_back (getTime ());
       
-      if (1==0){ // live capture - probably doesn't work anymore, left in here for comparison:
-	bool has_frame = evaluation_ptr_ ? evaluation_ptr_->grab(i, depth) : capture_.grab (depth, rgb24);      
-	if (!has_frame)
-	{
-	  cout << "Can't grab" << endl;
-	  break;
-	}
-
-	depth_device_.upload (depth.data, depth.step, depth.rows, depth.cols);
-	if (integrate_colors_)
-	    image_view_.colors_device_.upload (rgb24.data, rgb24.step, rgb24.rows, rgb24.cols);
-	
-	{
-	  SampledScopeTime fps(time_ms, i);
-	
-	  //run kinfu algorithm
-	  if (integrate_colors_)
-	    has_image = kinfu_ (depth_device_, image_view_.colors_device_);
-	  else
-	    has_image = kinfu_ (depth_device_);                  
-	}
-      }else{ //simulate:
-
-	cout << " color: " << integrate_colors_ << "\n"; // integrate_colors_ seems to be zero
-	depth_device_.upload (depth_sim.data, depth_sim.step, depth_sim.rows, depth_sim.cols);
-	if (integrate_colors_){
-	    image_view_.colors_device_.upload (rgb24_sim.data, rgb24_sim.step, rgb24_sim.rows, rgb24_sim.cols);
-	}
-	
-	tic_toc.push_back (getTime ());
-	
-	{
-	  SampledScopeTime fps(time_ms, i);
-	  //run kinfu algorithm
-	  if (integrate_colors_)
-	    has_image = kinfu_ (depth_device_, image_view_.colors_device_);
-	  else
-	    has_image = kinfu_ (depth_device_);                  
-	}
-	
-      }
+      /*if (1==0){ // live capture - probably doesn't work anymore, left in here for comparison:
```

Comment:
Just erase all things you commented in this file. Worst case scenario we have version control to go back in history.